src/SFA.DAS.EmploymentCheck.Functions.UnitTests/AzureFunctions/Activities/GetDbLearnerNiNumbersActivityTests/WhenCallingGet.cs

```diff
@@ -0,0 +1,68 @@
+﻿using AutoFixture;
+using MediatR;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmploymentCheck.Functions.Application.Models;
+using SFA.DAS.EmploymentCheck.Functions.AzureFunctions.Activities;
+using SFA.DAS.EmploymentCheck.Functions.Mediators.Queries.GetDbNiNumbers;
+using System.Linq;
+using System.Threading;
+using Models = SFA.DAS.EmploymentCheck.Functions.Application.Models;
+
+namespace SFA.DAS.EmploymentCheck.Functions.UnitTests.AzureFunctions.Activities.GetDbLearnerNiNumbersActivityTests
+{
+    public class WhenCallingGet
+    {
+        Fixture _fixture;
+        private Mock<IMediator> _mediator;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _fixture = new Fixture();
+            _mediator = new Mock<IMediator>();
+        }
+
+        [Test]
+        public void Then_The_LearnerNiNumbers_Are_Returned()
+        {
+            // Arrange
+            var employmentCheckBatch = _fixture.CreateMany<Models.EmploymentCheck>(1).ToList();
+            var learnerNiNumbers = _fixture.CreateMany<LearnerNiNumber>(1).ToList();
+            var sut = new GetDbLearnerNiNumbersActivity(_mediator.Object);
+            var queryResult = new GetDbNiNumbersQueryResult(learnerNiNumbers); // can't use _fixture, the LearnerNiNumbers property in GetDbNiNumbersQueryResult doesn't have a 'setter'
```

Comment:
This comment isn't needed, I understand why it's here but this kind of justification should be a response to a PR comment, not a pre-emptive code comment.

Reply:
Removed comment.